src/components/Dashboard/index.js

```diff
@@ -6,14 +6,14 @@ import {
   checkOrgHandleExists,
   checkUserHandleExists,
   clearProfileEditError,
-  setUpInitialData
+  setUpInitialData,
 } from "../../store/actions";
 import {
   GlobalOutlined,
   UserAddOutlined,
   AppstoreAddOutlined,
   AppstoreOutlined,
-  IeOutlined
+  IeOutlined,
```

Comment:
Follow Material-UI import guidelines as mentioned in https://github.com/scorelab/Codelabz/pull/75#discussion_r638543643.